components/settings/actions/SaveFile.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import Checkbox from 'material-ui/Checkbox';
+import TextField from 'material-ui/TextField';
+import {List, ListItem} from 'material-ui/List';
+import { blue500 } from 'material-ui/styles/colors';
+import Card, {CardHeader, CardText} from 'material-ui/Card';
+import Utils from '../../../Utils';
+
+export default class SaveFile extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            actionExecuted: false,
+            fileName: 'output',
+            netParams: true,
+            simConfig: true,
+            simData: true,
+            netCells: true
+        }
+        this.options = [
+            {label: 'High level specs.', label2: 'netParams', state: 'netParams'},
+            {label: 'High level specs.', label2: 'simConfig', state: 'simConfig'},
+            {label: 'Cells', label2: 'Instanciated Network cells', state: 'netCells'},
+            {label: 'Data', label2: 'Spikes, traces, etc.', state: 'simData'}
+        ]
+    }
+
+    componentDidMount () {
+        Utils.sendPythonMessage('netpyne_geppetto.doIhaveInstOrSimData', [])
+            .then(response => {
+                if (response.constructor.name=='Array') {
```

Comment:
that is checking if the user has instantiated a model before and if the user has simulation data from a previous simulation. It is returning an array with [ true/false, true/false].  So, I can use an object instead of an array?